app/src/androidTest/java/net/skyscanner/backpack/compose/barchart/BpkBarchartTests.kt

```diff
@@ -31,6 +36,9 @@ import kotlin.math.roundToInt
 @RunWith(AndroidJUnit4::class)
 class BpkBarchartTests : BpkSnapshotTest() {
 
+  @get:Rule
+  val composeTestRule = createAndroidComposeRule<AppCompatActivity>()
```

Comment:
by using a rule here this activity always gets started, no matter if the test actually gets run by the `assumeVariant` check. `createEmptyComposeRule` allows you to start the activity manually later